DESCRIPTION

```diff
@@ -80,5 +80,5 @@ Remotes:
     mrc-ide/dust,
     mrc-ide/eigen1,
     mrc-ide/mcstate,
-    mrc-ide/odin,
-    mrc-ide/odin.dust
```

Comment:
This will be removed shortly, but please depend on at least 0.3.6 in the DESCRIPTION and the generation script